pallets/asset-registry/src/lib.rs

```diff
@@ -162,27 +184,34 @@ pub mod pallet {
 	pub type LocationAssets<T: Config> =
 		StorageMap<_, Blake2_128Concat, T::AssetNativeLocation, T::AssetId, OptionQuery>;
 
-	#[pallet::storage]
-	#[pallet::getter(fn asset_metadata)]
-	/// Metadata of an asset.
-	pub type AssetMetadataMap<T: Config> =
-		StorageMap<_, Twox64Concat, T::AssetId, AssetMetadata<BoundedVec<u8, T::StringLimit>>, OptionQuery>;
```

Comment:
as I cannot comment on non-changed code, I will put the comment here

as it is permissionless now - for the Assets storage, we need to change the Hasher type to Blake2 instead of Twox64.

Same for assetlocations. or basically everywhere. 

this might need to be considered in migration though.